### ISSUE TITLE
es6 to es5

### ISSUE THREAD

**Issue by DrAugus** [cocos/cocos-engine#16697]
### Use Case

creator version: 3.8.1

### Problem Description

now does packaging automatically go from es6 to es5, is there any option to disable the transfer?

e.g.

```ts
let totalWeight: Map<number, number[]> = new Map<number, number[]>();
totalWeight.set(1, []);
totalWeight.set(2, []);
totalWeight.set(3, []);
const maxWeight: number = Math.max(...totalWeight.keys());
```
maxWeight will be **-Infinity**

### Alternatives Considered

ban es6 to es5

**Comment by dumganhar** [comment]
@DrAugus From v3.8.2, you could set the browserslist target for web platforms.
The target could be `supports es6-module` if you wish to use ES6 output. For more about the `target`, you could refer to https://github.com/browserslist/browserslist

<img width="450" alt="企业微信截图_c51ce2f9-bf67-430d-a7b0-99cf3d22ae23" src="https://github.com/cocos/cocos-engine/assets/493372/0c719159-2d53-44bc-9f66-26dfb6604b6d">

v3.8.2 is still in beta state and will be released before 2024.2.9.
You could try the beta version:

https://download.cocos.org/CocosCreator/v3.8.2/CocosCreator-v3.8.2-mac-011206.zip
https://download.cocos.org/CocosCreator/v3.8.2/CocosCreator-v3.8.2-win-011206.zip

**Comment by dumganhar** [comment]
I close this issue now. If you still get issues, feel free to open it again.
